mixer/adapter/stackdriver/config/config.proto

```diff
@@ -40,223 +40,177 @@ option (gogoproto.gostring_all) = false;
 
 // Configuration format for the `stackdriver` adapter.
 message Params {
-    // Endpoint URL to send stackdriver data - leave empty to use the
-    // StackDriver SDK's default value (monitoring.googleapis.com).
-    string endpoint = 1;
-
-    // GCP Project to attach metrics to.
-    string project_id = 2;
-
-    // This adapter batches the data it sends to Stackdriver; we will push to stackdriver every push_interval.
-    // If no value is provided we default to once per minute.
-    google.protobuf.Duration push_interval = 3 [(gogoproto.nullable)=false, (gogoproto.stdduration) = true];
```

Comment:
Why are we eschewing this config?  Do we believe that this was not being customized at all?